modules/kubernetes-addons/nvidia-device-plugin/locals.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+  name    = "nvidia-device-plugin"
+  version = "0.12.3"
+
+  default_helm_config = {
+    name             = local.name
+    chart            = local.name
+    repository       = "https://nvidia.github.io/k8s-device-plugin"
+    version          = local.version
+    namespace        = local.name
+    description      = "nvidia-device-plugin Helm Chart deployment configuration"
```

Comment:
I think we need to add the following as a default value
```hcl
    create_namespace = true
```

Reply:
This is already added, next line :)